index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="./css/stylesheet.css" />
+    <title>Home</title>
+  </head>
+  <body onload="showSpeakerSection()">
+    <header class="desktop-header">
+      <div class="social-media"></div>
```

Comment:
- [OPTIONAL] As making our design identical to the original design is one of the requirements of this project. kindly consider adding the top black colored navbar that you can see in the following picture to your design. This requirement was missed by the previous reviewer, so I will highly recommend you to fix this.
original design
![qw](https://user-images.githubusercontent.com/96683843/227277904-787bea67-4629-4eb2-a6c1-e56790a1a67f.jpg)